integration/cli_test.go

```diff
@@ -37,11 +36,12 @@ func TestMain(m *testing.M) {
 	}
 	os.Exit(m.Run())
 }
-
+*/
 func TestVersionCommand(t *testing.T) {
 	if testing.Short() {
 		t.Skip("skipping test in short mode.")
 	}
+	t.Skip("skipping - this test makes little or no sense")
```

Comment:
disabling this test - it doesn't have any value.

Reply:
if it doesn't make any sense, delete it 😄